.stylelintrc

```diff
@@ -1,9 +1,109 @@
 {
-  "extends": "stylelint-config-recommended",
+  "extends": ["stylelint-config-recommended", "stylelint-config-standard-scss"],
+  "plugins": [
+    "stylelint-scss"
+  ],
   "rules": {
-    "no-duplicate-selectors": null,
-    "font-family-no-duplicate-names": null,
-    "no-descending-specificity": null,
-    "selector-type-no-unknown": null
+    "alpha-value-notation": "number",
```

Comment:
Are all these rules specific to `auro-button` if not (just an idea) this might be a great addition to add the `auro-library` to standardize our styleling styles

Reply:
Please see my comment below